### ISSUE TITLE
Wrong alignment of switch when "box-sizing: border-box;" is not set on switch element

### ISSUE THREAD

**Issue by ul-gh** [adamculpepper/toggle-switchy#21]
Hi Adam,

thank you for your library!

I had a hard time getting the alignment of the switch button correct with the example code.

Running the example on a clean homepage without any CSS framworks included, the buttons were displayed as in this image:
![switch_wrong_alignment](https://user-images.githubusercontent.com/16521333/90825879-dbe3c580-e339-11ea-9ff2-5384b55d2457.png)
 
Strangely, the buttons are displayed correctly when just including bootstrap.css as seems to be the case for the examples I found on Google: 
![switch_correct_alignment](https://user-images.githubusercontent.com/16521333/90826527-ddfa5400-e33a-11ea-8b51-c409fee1add4.png)

The reason I found just now, these framworks apply the global style "box-sizing: border-box;" which is not a browser default at least on Firefox and current Chrome versions.

I fixed this by adding the global style, but this is also fixed by adding it to the switch element CSS as with the following patch:
```
diff --git a/toggle-switchy.css b/toggle-switchy.css
index dd6ac4a..e6f7acb 100644
--- a/toggle-switchy.css
+++ b/toggle-switchy.css
@@ -34,7 +34,7 @@ CORE STYLES BELOW - NO TOUCHY
 .toggle-switchy > input[disabled] + .toggle {opacity:0.5;}
 .toggle-switchy > input[disabled] + .toggle:hover {cursor:not-allowed;}
 .toggle-switchy > input + .toggle {width:100%; height:100%; margin:0; cursor:pointer;}
-.toggle-switchy > input + .toggle > .switch {display:block; height:100%; position:absolute; right:0; z-index:3;}
+.toggle-switchy > input + .toggle > .switch {box-sizing:border-box; display:block; height:100%; position:absolute; right:0; z-index:3;}
```

HTH, best wishes.

**Comment by adamculpepper** [comment]
@ul-gh I've been meaning to test the toggles on a non-bootstrap website. Thank you so much for finding the fix for this! I likely would have wasted a bunch of time on this one - I wouldn't have thought the `box-sizing` to be the culprit here.

I can push a fix or you can so you get credit for the contribution. Just let me know either way. :)

Thanks again!

**Comment by ul-gh** [comment]
Adam Culpepper wrote:
> I can push a fix or you can so you get credit for the contribution.
> Just let me know either way. :)

Never mind, please go ahead, I don't want any credit, I am glad that it
works for me..

**Comment by adamculpepper** [comment]
Fixed and pushed to release version 1.14

Thanks again @ul-gh ! :)
